### PR TITLE
Add 2018 June meetup and ThoughtWorks location

### PR DESCRIPTION
**Pull request by killercup**

### CODE REVIEW

File: _posts/2018-06-27-thoughtworks.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    'Share Secrets Safely' scratches a particular itch as it provides the tooling required to use GPG in teams with great user experience, while providing all tooling necessary to introduce said shared secrets into their final destination without them ever touching disk. I will particularly highlight the way the project is structured, and tested, showing that 'journey' level testing is now affordable thanks to Rust.
+- title: "The HUB"
+  speaker:
+    name: "Niko Hellwig"
+    github: nhellwig
+  abstract: |
+    A central entry point to your current project
+---
+Our next Meetup will be on 2018-06-27, 19:15 CET. We are very happy to be
+invited to a **new location**: The meetup will be at ThoughtWorks office in Cologne
```
Comment (Florob) [inline _posts/2018-06-27-thoughtworks.md]: "**the** ThoughtWorks office"?

File: _posts/2018-06-27-thoughtworks.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+(Lichtstraße 43i -- just 400m from the C4).
+
+In addition to the new location, we're also quite excited to have two talks
+prepared for you! the first one, by Sebastian Thiel is an introduction to "Share
+Secrets Safely", and present a tool to share secrets within teams to avoid
+plain-text secrets. In the second talk, Niko Hellwig will talk about "HUB", a
+central entry point to your current project.
```
Comment (Florob) [inline _posts/2018-06-27-thoughtworks.md]: Maybe clarify how these relate to Rust?

Comment (killercup) [reply _posts/2018-06-27-thoughtworks.md]: good point

**Review by Florob — APPROVED** [review]
LGTM

**Pull request merged**